README.md

```diff
@@ -198,6 +198,7 @@ Variables:
 * `\i` : Cloud Spanner Instance ID
 * `\d` : Cloud Spanner Database ID
 * `\t` : In transaction
+* `\c` : Current time in RFC3339 format
```

Comment:
I couldn't find good variable name,  `\t` (for time) is used for `In transaction`, `\d` (for date) is used for `Database ID`,
I choose `\c` (for current time).